deployments/daemon-set/nginx-ingress.yaml

```
@@ -48,12 +48,22 @@ spec:
          #  memory: "1Gi"
         securityContext:
           allowPrivilegeEscalation: true
+          readOnlyRootFilesystem: true
```

```suggestion
          readOnlyRootFilesystem: false
```
Users will set this back to `true` when they uncomment the volumes below.
This same update can be made to `deployments/deployment/nginx-ingress.yaml`, `deployments/deployment/nginx-plus-ingress.yaml` and `deployments/deployment/nginx-plus-ingress.yaml`